kong/db/strategies/off/plugins.lua

```diff
@@ -0,0 +1,46 @@
+local null = ngx.null
+
+local Plugins = {}
+
+function Plugins:select_by_ca_certificate(ca_id, limit, plugin_names)
```

Comment:
It's likely that I am missing something, but I wonder if dao/strategy code for `off` is truly necessary here? I recall that `:delete()` is [special-cased to return an error](https://github.com/Kong/kong/blob/a382576530b7ddd57898c9ce917343bddeaf93f4/kong/db/strategies/off/init.lua#L310-L316) in dbless mode, which in theory should make `:select_by_ca_certificate()` unreachable assuming we're only using it to validate `:delete()`.

Reply:
Right, the code under `off` is not being used. I added them just for completeness. If you guys are more inclined to remove it, I will.